### PR TITLE
net-p2p/biglybt: disable SWTUpdateChecker

### PR DESCRIPTION
**Pull request by vaukai**
Closes: https://bugs.gentoo.org/889032
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

**Comment by Flowdalic** [comment]
Thanks. This requires a version bump.

**Comment by vaukai** [comment]
> Thanks. This requires a version bump.

rbumped.

File: net-p2p/biglybt/files/biglybt-3.2.0.0-disable-shared-plugins.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
+diff --git a/uis/src/com/biglybt/ui/swt/plugininstaller/IPWFilePanel.java b/uis/src/com/biglybt/ui/swt/plugininstaller/IPWFilePanel.java
```
[the remark this answers is not in the document]
Comment (vaukai) [reply net-p2p/biglybt/files/biglybt-3.2.0.0-disable-shared-plugins.patch]: Done.

File: net-p2p/biglybt/files/biglybt-3.2.0.0-disable-SWTUpdateChecker.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- a/uis/src/com/biglybt/ui/swt/updater2/SWTUpdateChecker.java
++++ b/uis/src/com/biglybt/ui/swt/updater2/SWTUpdateChecker.java
+@@ -65,7 +65,7 @@ public class SWTUpdateChecker implements UpdatableComponent
+   public static void
+   initialize()
+   {
+-  	PluginInitializer.getDefaultInterface().getUpdateManager().registerUpdatableComponent(new SWTUpdateChecker(),true);
++//  	PluginInitializer.getDefaultInterface().getUpdateManager().registerUpdatableComponent(new SWTUpdateChecker(),true);
```
[the remark this answers is not in the document]
Comment (vaukai) [reply net-p2p/biglybt/files/biglybt-3.2.0.0-disable-SWTUpdateChecker.patch]: Done

File: net-p2p/biglybt/biglybt-3.2.0.0-r1.ebuild [hunk 3]
```diff
@@ -53,13 +53,21 @@ DOCS=(
...
 
 src_prepare() {
 	default
-	# This directory would break compilation with jdk >= 11
-	# "error: package sun.net.spi.nameservice does not exist"
-	rm -r core/src/com/biglybt/core/util/spi || die
+	# AENameServiceDescriptor fails to compile with jdk >= 11
+	# https://github.com/BiglySoftware/BiglyBT/commit/de4f7805ce73684ed8220ea63952abb9dd4a9e3a
+	local vm_version="$(java-config -g PROVIDES_VERSION)"
+	if ver_test "${vm_version}" -ge 11; then
```
[the remark this answers is not in the document]
Comment (vaukai) [reply net-p2p/biglybt/biglybt-3.2.0.0-r1.ebuild]: Done, lifting to min java 11

**Pull request closed without merging**